service/routes.py

```diff
@@ -29,3 +29,20 @@ def index():
 ######################################################################
 
 # Place your REST API code here ...
+
+@app.route("/inventory/<int:id>", methods=["DELETE"])
+def delete_inventory(id):
+    app.logger.info("Request to delete inventory with ID %s", id)
+    inventory = Inventory.find(id)
+    if inventory:
+        condition = inventory.condition
+        if condition == Condition.NEW:
```

Comment:
What's this check for? I thought you wouldn't care about the condition of an item if you're deleting it?

Reply:
resolved